src/firebase.js

```diff
@@ -26,13 +26,13 @@ if (window.location.hostname === "localhost") {
   db.useEmulator("localhost", 8080);
 }
 
-// Add patient data and trial data to db
-const createPatient = (patientId, studyId, startDate, timestamp) => {
-  // return promise with value true if patient and study id match, false otherwise
+// Add participant data and trial data to db
+const validateParticipant = (participantId, studyId, startDate, timestamp) => {
```

Comment:
this does more than validate, I would keep it as `createParticipant` or `initParticipant`